### DISCUSSION TITLE
Test is executed fine from Xcode but not through patrol cli.

### DISCUSSION THREAD

**Discussion by Piyu-sh-arma** [leancodepl/patrol#1023] [Q&A]
Hi, I followed steps mentioned in setup guide for iOS. Finally when I did Product -> Test in Xcode, I can see that test was executed. but when I used command **Patrol test --target integration_test/example_test.dart** in terminal. I get following output and it gets stuck there keep wait for app launch.

No device specified, using the first one (iPhone X)
• Building app with entrypoint example_test.dart for iOS device...
✓ Completed building app with entrypoint example_test.dart for iOS device (88.3s)
• Running app with entrypoint example_test.dart for iOS device on device iPhone X...
Waiting for ios-deploy to launch the app is taking unusually long time (30s)
Waiting for ios-deploy to launch the app is taking unusually long time (30s)
Waiting for ios-deploy to launch the app is taking unusually long time (30s)

**Comment by bartekpacia** [comment]
Hi @Piyu-sh-arma, this looks like a duplicate of #949. So sorry about the problems!

See also #984.

**Reply by Piyu-sh-arma** [reply]
Thanks for clarification :)
